translate/src/modules/resourceprogress/components/ProgressChart.tsx

```diff
@@ -49,6 +49,10 @@ export default class ProgressChart extends React.Component<Props> {
       this.props.stats;
     const dpr = window.devicePixelRatio || 1;
     const canvas = this.canvas.current;
+    if (!canvas) return;
+
+    const context = canvas.getContext('2d');
+    if (!context) return;
```

Comment:
Just a spoon full of sugar...

```suggestion
    const context = canvas?.getContext('2d');
    if (!context) return;
```